### PR TITLE
Updated Replanter to harvest tea leaves from green tea bushes

### PR DESCRIPTION
**Pull request by schmitzrj**
Also includes a fix on a divide by zero error when the player is at full farming level 10, previously fixed by mizzion, but I couldn't find that code, so I fixed it myself.

### CODE REVIEW

File: Replanter/Replanter.csproj [hunk 1]
```diff
@@ -3,7 +3,7 @@
   <PropertyGroup>
     <AssemblyName>Replanter</AssemblyName>
     <RootNamespace>Replanter</RootNamespace>
-    <Version>1.1.1</Version>
+    <Version>1.1.3-unofficial.1-jacquesdereims</Version>
```
Comment (Pathoschild) [inline Replanter/Replanter.csproj]: (This shouldn't be in the pull request, since it would add the unofficial version number to the official repo.)

Comment (schmitzrj) [reply Replanter/Replanter.csproj]: Reverted version number to take out unofficial version, sorry!

**Comment by Pathoschild** [comment]
Thanks! This is included in newer pull request #16; can you close this one to avoid confusion?

**Comment by schmitzrj** [comment]
No problem, thanks for merging it in.

**Pull request closed without merging**